### PR TITLE
Fix linear regression

### PR DESCRIPTION
**Pull request by kevinthesun**
Sometimes linear regression tutorial returns 'nan' due to hardcoded training data. Use uniform random data and decrease learning rate.

### CODE REVIEW

File: docs/tutorials/python/linear-regression.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ how to encode a dataset into an iterator that MXNet can use. The data used in th
 
 ```python
 #Training data
-train_data = np.array([[1,2],[3,4],[5,6],[3,2],[7,1],[6,9]])
-train_label = np.array([5,11,17,7,9,24])
+train_data = np.random.uniform(0, 1, [100, 2])
```
Comment (madjam) [inline docs/tutorials/python/linear-regression.md]: Keep in mind that target audience for these tutorials (especially LR) is beginners. This is a good opportunity to briefly mention about the importance of feature normalization.

**Comment by madjam** [comment]
Please change this to add input normalization as a separate step.
